.github/workflows/trivy-analysis.yml

```diff
@@ -0,0 +1,67 @@
+name: Trivy
+on:
+  workflow_dispatch:
+  # Uncomment this for testing purposes
+  #push:
+  #  branches: [ main ]
+  # Run the job twice a day  
+  schedule:
+    - cron: "0 0,12 * * *"
+
+jobs:
+  quarkus-dist:
+    name: Vulnerability scanner for Quarkus distribution images
+    runs-on: "ubuntu-18.04"
+    steps:
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@296212627a1e693efa09c00adc3e03b2ba8edf18
+        with:
+          image-ref: 'quay.io/keycloak/keycloak:nightly'
```

Comment:
Should we run those actions right after the nightly builds?
https://github.com/keycloak-rel/keycloak-rel/blob/main/.github/workflows/nightly.yml

cc. @stianst